packages/core/src/statements/dedupe.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+/**
+ * Take `Object`, and check that the object identifier has not already been used previously
+ *
+ * @param {String} [path = uri] path containing the object Identifier
+ * @param {Boolean} [ignore = false] Just ignore duplicate object
+ * @returns {Object}
+ */
+export default async function identify(data, feed) {
```

Comment:
Shouldn't that function be named `dedupe`, and not `identify` ?

Reply:
of course, it will be better for the doc ...
![image](https://github.com/Inist-CNRS/ezs/assets/15609/7a82a1d1-fec9-432b-bd18-8cd218ed8eb9)
and I will correct another similar error